doc/developer_guide/index.md

```diff
@@ -55,49 +52,44 @@ checkout* for the remainder of this document.
 
 (dev-guide-installing-dependencies)=
 
-## Installing Dependencies
+### Installing Dependencies
 
 hvPlot requires many additional packages for development and
 testing.
 
-### Conda Environments
+::::{tab-set}
 
-Create an empty conda environment with the name that you prefer, here we've
-chosen hvplot_dev. Activate and configure its channels to only use
-`pyviz/label/dev` and `conda-forge`. The former is used to install the
-development versions of the other HoloViz packages, such as HoloViews or Panel.
+:::{tab-item} pip
+:sync: pip
 
-```sh
-conda create -n hvplot_dev
-conda activate hvplot_dev
-conda config --env --append channels pyviz/label/dev --append channels conda-forge
-conda config --env --remove channels defaults
-```
+Start by creating a virtual environment. For instance, on Linux/MacOs:
 
-Since hvPlot interfaces with a large range of different libraries the
-full test suite requires a wide range of dependencies. To make it
-easier to install and run different parts of the test suite across
-different platforms hvPlot uses a library called `pyctdev` to make things
-more consistent and general. Specify also the desired Python version you want
-to base your environment on.
+```
+python -m venv .venv
+source .venv/bin/activate
+```
 
-You will need to pick a Python version. The best practice is to choose the minimum version
-currently supported by hvPlot on the main development branch. If you cannot get the minimum
-version installed, then try with a more recent version of Python.
+Install the test dependencies:
 
-```sh
-conda install python=3.x pyctdev
+``` bash
+pip install -e '.[tests, examples-tests, geo, hvdev, hvdev-geo]'
```

Comment:
You could split this into two commands, one with and one without geo dependencies. 

Another thing that could be mentioned is that hvdev pulls in dev versions of other holoviz packages.

Reply:
> You could split this into two commands, one with and one without geo dependencies.

I think the developer guide will be refined over time, that's just the first pass, I'm fine with how it is at the moment. In a few months when hvPlot drops support of Python 3.8 hopefully the non-geo/geo stuff can be removed.

> Another thing that could be mentioned is that hvdev pulls in dev versions of other holoviz packages.

Same, I think some more in-depth explanation about the relationship between hvPlot and other packages, HoloViews in particular, is needed for hvPlot contributors.